tests/unit/steps/test_generate_lz4.py

```diff
@@ -0,0 +1,45 @@
+import os
+import time
+import unittest
+
+from tests.unit.unittest_helpers import relative_file_path
+
+from idseq_dag.steps.generate_lz4 import PipelineStepGenerateLZ4
+
+INPUT_FILE = relative_file_path(__file__, 'dummy testfile.txt')
+OUTPUT_FILE = INPUT_FILE + '.lz4'
+INPUT_FILE_GZIP = INPUT_FILE + '.gz'
+OUTPUT_FILE_GZIP = INPUT_FILE_GZIP + '.lz4'
+
+class TestPipelineStepGenerateLZ4(unittest.TestCase):
+
+    def setUp(self):
+        self.step = PipelineStepGenerateLZ4(
+            name='test_generate_lz4',
+            input_files=[[INPUT_FILE, INPUT_FILE, INPUT_FILE_GZIP]],
+            output_files=[OUTPUT_FILE, OUTPUT_FILE, OUTPUT_FILE_GZIP],
+            output_dir_local='',
+            ref_dir_local='',
+            output_dir_s3='',
+            additional_files={},
+            additional_attributes={},
+            step_status_lock=None,
+            step_status_local=''
+        )
+
+    def test_get_command(self):
+        command = self.step.get_command(INPUT_FILE)
+        self.assertEqual('lz4', command.cmd)
+        self.assertSequenceEqual(
+            ['-9', '-f', INPUT_FILE],
+            command.args
+        )
+
+    def test_run(self):
+        self.step.run()
```

Comment:
I'm was wondering if we should put tests like this `test_run` in a separate test folder (integration?), considering it is actually invoking an external dependency (therefore it is not a unit test anymore), but this might be a much broader discussion. I guess we can leave it this way for now.

Reply:
Taken with your comment above, I decided to remove the `test_run` integration part of the unit test. 

To do a proper integration test, I think we should create a test dag json, such as `examples/accession2taxid_dag_test.json`.